vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -348,7 +350,7 @@ def set_relationship_on_hosts(hosts)
       #
 
       def normalize_blank_property_num(property)
-        property.try(:to_i)
+        property.try(:to_i) || 0
```

Comment:
We may have different definitions of "work" ;)

If some non-nil, non-string, non-numeric value turns up, that doesn't respond to `to_i`, I'd much rather see an "I don't know what this is" error than quietly assume it's another way of spelling zero.